packages/cgb-scripts/package.json

```diff
@@ -45,7 +56,7 @@
 		"eslint-plugin-jsx-a11y": "^6.0.3",
 		"eslint-plugin-react": "^7.5.1",
 		"eslint-plugin-wordpress": "^0.1.0",
-		"extract-text-webpack-plugin": "^3.0.2",
+		"extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
Does this work in the same way? I mean is it backward compatible for so many plugins built on top of create-guten-block?

Reply:
This one was hard to determine as that package appears to be outdated and unmaintained. However, if I remember correctly, v4.0.0-beta.0 is the only one that works with webpack 4.

Ultimately, I believe that removing the extract-text-webpack-plugin dependency and replacing it with mini-css-extract plugin is the better route to go.

Reply:
I am open to replace it, if you'd like to take charge here. 

Two options:

1. https://quantizd.com/webpack-4-extract-css-with-mini-css-extract-plugin/
2. https://stackoverflow.com/questions/50976979/output-2-or-more-css-files-with-mini-css-extract-plugin-in-webpack

I'm siding with the second option if the first one doesn't work. Go ahead and try that?

Reply:
@ahmadawais it looks like @asharirfan's PR addresses this and includes the mini-css-extract-plugin work:

https://github.com/ahmadawais/create-guten-block/pull/166

Should I merge that work into my branch and continue testing from there?

Reply:
#166 is draft PR and WIP. You can start your own separate one, I test #166 and it was not working.